simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -2634,7 +2615,7 @@
         "mainColor": "blue",
         "name": "San Francisco Public Library",
         "needsAuth": true,
-        "pinKeyboard": "Number pad",
+        "pinKeyboard": "Default",
```

Comment:
Usually these libraries go from 'default' for the PIN keyboard to 'number pad'. San Francisco is going the other way; the original change was a mistake.